lib/Runtime/ByteCode/ByteCodeEmitter.cpp

```diff
@@ -8412,7 +8422,7 @@ void EmitObjectInitializers(ParseNode *memberList, Js::RegSlot objectLocation, B
             memberList = memberList->AsParseNodeBin()->pnode2;
         }
 
-        if (memberList->AsParseNodeBin()->pnode1->nop != knopComputedName && !hasComputedName)
+        if (memberList->nop != knopEllipsis && memberList->AsParseNodeBin()->pnode1->nop != knopComputedName && !hasComputedNameOrSpread)
```

Comment:
Why is this one `memberList->nop` rather than `memberList->AsParseNodeBin()->pnode1->nop` like in most other places?

Reply:
never mind, figured it out :)

---
In reply to: [192910053](https://github.com/Microsoft/ChakraCore/pull/5268#discussion_r192910053) [](ancestors = 192910053)